### PR TITLE
Added a little chat logout announcer

### PR DESCRIPTION
**Pull request by widdwecat**
I may have changed something that I shouldn't have, but here it is I guess..

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/LogoutLogger.kt [hunk 1]
```diff
@@ -44,9 +45,10 @@ object LogoutLogger : Module() {
                 if (mc.connection!!.getPlayerInfo(name) != null) continue
                 if (print.value) MessageSendHelper.sendChatMessage("$name logged out at ${pos.asString()}")
                 if (saveToFile.value) WaypointManager.add(pos, "$name Logout Spot")
+                if (announcer.value) mc.player.sendChatMessage( "KAMI BLUE on top! EZ Log, $name")
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/LogoutLogger.kt:48]: I don't understand the point of this? Line 46 does the same thing but more useful

Comment (Xiaro) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/LogoutLogger.kt:48]: I think `mc.player.sendChatMessage` sends a message to the server. They were meant to do `MessageSendHelper.sendServerMessage`

Comment (Xiaro) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/LogoutLogger.kt:48]: This is basically AutoEz but for logout

Comment (l1ving) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/LogoutLogger.kt:48]: Yea that's still won't fix, it's essentially useless. I'd rather someone make a plugin

Comment (GoodPro712) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/LogoutLogger.kt:48]: it's supposed to say to chat "ez log" when someone logs near them to be toxic

Comment (Xiaro) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/LogoutLogger.kt:48]: ^

**Pull request closed without merging**